src/main/resources/l10n/JabRef_de.properties

```diff
@@ -1696,7 +1696,7 @@ Select_file_type\:=Dateityp_wählen\:
 These_files_are_not_linked_in_the_active_database.=Diese_Dateien_sind_in_der_aktiven_Datenbank_nicht_verlinkt.
 Entry_type_to_be_created\:=Eintragstyp,_der_erstellt_werden_soll\:
 Searching_file_system...=Dateisystem_wird_durchsucht...
-Importing_into_Database...=Importieren_in_die_Datei...
+Importing_into_Database...=Importieren_in_die_Datenbank...
```

Comment:
You should stay consistent here. If you decide for "Importiere" below, then you should also use here "Importiere".